### PR TITLE
Docs: fixed typo in Stardew Valley setup guide

### PR DESCRIPTION
**Pull request by benny-dreamly**
## What is this fixing or adding?

fixes typo 

## How was this tested?

👀

### CODE REVIEW

File: worlds/stardew_valley/docs/en_Stardew Valley.md [hunk 1]
```diff
@@ -124,6 +124,6 @@ List of supported mods:
 
 ## Multiplayer
 
-You cannot play an Archipelago Slot in multiplayer at the moment. There is no short-terms plans to support that feature.
+You cannot play an Archipelago Slot in multiplayer at the moment. There is no short-term plans to support that feature.
```
Comment (nicholassaylor) [inline worlds/stardew_valley/docs/en_Stardew Valley.md:127]:
```suggestion
You cannot play an Archipelago Slot in multiplayer at the moment. There are no short-term plans to support that feature.
```
Should also have a plural verb while we're at it.

**Review by nicholassaylor — CHANGES_REQUESTED** [review]

File: worlds/stardew_valley/docs/en_Stardew Valley.md (hunk 1 above)
Comment (benny-dreamly) [reply worlds/stardew_valley/docs/en_Stardew Valley.md:127]: yeah probably :)

**Review by nicholassaylor — APPROVED** [review]
LGTM

**Review by agilbert1412 — APPROVED** [review]

**Comment by agilbert1412** [comment]
Please request me as reviewer for Stardew PRs in the future

**Review by agilbert1412 — APPROVED** [review]

**Review by black-sliver — APPROVED** [review]

**Pull request merged**